doc/howto/storage_create_pool.md

```diff
@@ -8,6 +8,8 @@ To create a storage pool, use the following command:
 
     lxc storage create <pool_name> <driver> [configuration_options...]
 
+Unless specified otherwise, LXD sets up loop-based storage with a sane default size (20% of the free disk space, but at least 5 GiB and at most 30 GiB).
```

Comment:
I believe "sane" is frowned upon for inclusive language, as to relates to mental health (the counter being insane).

Reply:
Oh right - I wonder why the inclusive language check doesn't catch it. Probably that one isn't in the official word list yet ...